assignments/lambda-classes.js

```diff
@@ -1 +1,254 @@
 // CODE here for your Lambda Classes
+
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+    }
+
+    speak() {
+        return (`Hello my name is ${this.name}, I am from ${this.location}.`);
+    }
+}
+
+const fred = new Person({
+    name: 'Fred',
+    location: 'Bedrock',
+    age: 37,
+    gender: 'male',
+  });
+
+  const bread = new Person({
+    name: 'bread',
+    location: 'Breadrock',
+    age: 27,
+    gender: 'female',
+  });
+
+  const lead = new Person({
+    name: 'lead',
+    location: 'Leadrock',
+    age: 17,
+    gender: 'non-binary',
+  });
+
+console.log(fred.speak());
+console.log(bread.speak());
+console.log(lead.speak());
+
+class Instructor extends Person {
+    constructor(instructorAttr) {
+        super(instructorAttr);
+        this.specialty = instructorAttr.specialty;
+        this.favLanguage = instructorAttr.favLanguage;
+        this.catchPhrase = instructorAttr.catchPhrase;
+    }
+
+    demo(subject) {
+        return (`Today we are learning about ${subject}.`);
```

Comment:
```suggestion
        return `Today we are learning about ${subject}.`;
```
For single-line returns you don't need to wrap your code in parentheses.